bin/data.py

```diff
@@ -50,16 +67,18 @@ class ExerciseConfig:
     forked_from: str = None
     contributors: List[str] = None
     language_versions: List[str] = None
+    source: str = None
+    source_url: str = None
 
     def __post_init__(self):
         if isinstance(self.files, dict):
             self.files = ExerciseFiles(**self.files)
-        for attr in ['authors', 'contributors', 'language_versions']:
+        for attr in ["authors", "contributors", "language_versions"]:
```

Comment:
Concept exercises also appear to have "source" and "source_url" properties.  Don't know if we want to check for/validate any that have those fields, or not. [ Link to current spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#full-example).